packages/react-server-cli/src/commands/start.js

```diff
@@ -80,11 +82,23 @@ const startHtmlServer = (serverRoutes, port, bindIp, httpsOptions, customMiddlew
 		started: new Promise((resolve, reject) => {
 			logger.info("Starting HTML server...");
 
-			let rsMiddlewareCalled = false;
-			const rsMiddleware = () =>  {
-				rsMiddlewareCalled = true;
-				reactServer.middleware(server, require(serverRoutes));
-			}
+				let rsMiddlewareCalled = false;
+				const rsMiddleware = () => {
+					rsMiddlewareCalled = true;
+
+					expressState.extend(server);
+
+					// parse cookies into req.cookies property
+					server.use(cookieParser());
+
+					// sets the namespace that data will be exposed into client-side
+					// TODO: express-state doesn't do much for us until we're using a templating library
+					server.set('state namespace', '__reactServerState');
+
+					server.use((req, res, next) => {
+						reactServer.middleware(req, res, next, require(serverRoutes));
+					});
+				};
```

Comment:
Hmmm... we just run `asini run test` in CI.  So each package is responsible for making sure that linting is part of its test script.  It _looks_ like the CLI's package.json should be running tests as part of the `gulp test` phase of its test script.

Wonder where that's breaking... 🤔